features/support/ui/pages/money_manager.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManager < UI::Page
+    set_url '/en/tools/money-manager/'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
+    element :received_first_payment_false,
+            '#questionnaire_received_first_payment_false'
```

Comment:
The same thing applies here when it comes to de-coupling actual CSS implementations.